packages/adapter/src/utxo/bitcoincash/bitcoincash.ts

```diff
@@ -1,3 +1,4 @@
+import type { PublicKey } from '@shapeshiftoss/hdwallet-core'
```

Comment:
Wrap all responses `MaybeRpcResponse<PublicKeys>`. right now it looks like bitcoin is the only one

Reply:
0b776a653ba84cccd431e92eb36bb78c11ac756c